pymclevel/materials.py

```diff
@@ -775,22 +786,23 @@ def addBlock(self, blockID, blockData=0, **kw):
 pocketMaterials.DarkGreenWool = pocketMaterials[35, 13]
 pocketMaterials.RedWool = pocketMaterials[35, 14]
 pocketMaterials.BlackWool = pocketMaterials[35, 15]
+
 pocketMaterials.Flower = pocketMaterials[37, 0]
 pocketMaterials.Rose = pocketMaterials[38, 0]
 pocketMaterials.BrownMushroom = pocketMaterials[39, 0]
 pocketMaterials.RedMushroom = pocketMaterials[40, 0]
 pocketMaterials.BlockofGold = pocketMaterials[41, 0]
 pocketMaterials.BlockofIron = pocketMaterials[42, 0]
+
 pocketMaterials.DoubleStoneSlab = pocketMaterials[43, 0]
 pocketMaterials.DoubleSandstoneSlab = pocketMaterials[43, 1]
-pocketMaterials.DoubleWoodenSlab = pocketMaterials[43, 2]
 pocketMaterials.DoubleCobblestoneSlab = pocketMaterials[43, 3]
 pocketMaterials.DoubleBrickSlab = pocketMaterials[43, 4]
 pocketMaterials.StoneSlab = pocketMaterials[44, 0]
 pocketMaterials.SandstoneSlab = pocketMaterials[44, 1]
-pocketMaterials.WoodenSlab = pocketMaterials[44, 2]
```

Comment:
I dont know if you should delete this line and `[43, 2]`.
These are the IDs for the old slabs that still appear in worlds that were generated 2-3 years ago.